ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py

```diff
@@ -0,0 +1,970 @@
+#!/usr/bin/python
+
+# Copyright 2014 iNuron NV
+#
+# Licensed under the Open vStorage Modified Apache License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.openvstorage.org/license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Open vStorage Health Check module
+"""
+
+import os
+import grp
+import stat
+import glob
+import time
+import psutil
+import signal
+import socket
+import threading
+import subprocess
+from pwd import getpwuid
+from ovs.dal.lists.vpoollist import VPoolList
+from ovs.extensions.generic.system import System
+from ovs.dal.lists.servicelist import ServiceList
+from ovs.dal.lists.pmachinelist import PMachineList
+from ovs.dal.lists.mgmtcenterlist import MgmtCenterList
+from ovs.lib.storagerouter import StorageRouterController
+from ovs.extensions.healthcheck.utils.extension import Utils
+from ovs.log.healthcheck_logHandler import HCLogHandler
+import volumedriver.storagerouter.storagerouterclient as src
+
+
+class OpenvStorageHealthCheck:
+    """
+    A healthcheck for the Open vStorage framework
+    """
+
+    def __init__(self, logging=HCLogHandler(False)):
+        """
+        Init method for Open vStorage health check module
+
+        @param logging: ovs.log.healthcheck_logHandler
+
+        @type logging: Class
+        """
+
+        self.module = 'openvstorage'
+        self.LOGGER = logging
+        self.utility = Utils()
+        self.service_manager = self.utility.serviceManager
+        self.machine_details = System.get_my_storagerouter()
+        self.machine_id = self.machine_details.machine_id
+        self.max_logsize = 500  # in MB
+
+        # list of packages on your local system
+        self.openvstorageTotalPackageList = ["openvstorage", "openvstorage-backend", "openvstorage-backend-core",
+                                             "openvstorage-backend-webapps", "openvstorage-core", "openvstorage-hc",
+                                             "openvstorage-sdm", "openvstorage-webapps", "openvstorage-test",
+                                             "alba", "volumedriver-base", "volumedriver-server", "nginx", "memcached",
+                                             "rabbitmq-server", "qemu-kvm", "virtinst", "openvpn", "ntp",
+                                             "swiftstack-node"
+                                             ]
```

Comment:
```]``` should be on the previous line, to use the same code style as the rest of the project. The same for all other places.